packages/flutter_tools/test/integration.shard/flutter_plugins_dependencies_test.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+import 'package:file/file.dart';
+
+import '../src/common.dart';
+import 'test_utils.dart';
+
+void main() {
+  late Directory tempDir;
+
+  setUp(() async {
```

Comment:
Here and below: unless the `await` keyword is used, `async` isn't needed. Where you probably see it across Dart unit tests is that a lot of the setup and teardown work is _often_ asynchronous, but you're explicitly using the synchronous variants (`*Sync*`).

That's totally fine, just wanted to explain why these did nothing.